### PR TITLE
fix(platform): combobox in mobile mode

### PR DESCRIPTION
**Pull request by dimamarksman**
#### Please provide a link to the associated issue.
Closes: https://github.com/SAP/fundamental-ngx/issues/6046

#### Please provide a brief summary of this pull request.
This PR fixes some mobile combobox errors 
#### Please check whether the PR fulfills the following requirements

- [x] the commit message follows the guideline:
https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
- [x] tests for the changes that have been done
- [x] all items on the PR Review Checklist are addressed :
https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
- [x] Run npm run build-pack-library and test in external application

Documentation checklist:
- [na] update `README.md`
- [na] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)
- [x] Documentation Examples
- [x] Stackblitz works for all examples

### CODE REVIEW

**Review by olshansky — APPROVED** [review]

**Review by platon-rov — APPROVED** [review]

File: libs/platform/src/lib/components/form/combobox/combobox-mobile/combobox/combobox-mobile.component.spec.ts [hunk 1]
```diff
@@ -17,43 +17,37 @@ describe('ComboboxMobileComponent', () => {
...
+        };
+        dialogApprove(): void {
+            //
+        }
+        dialogDismiss(backup: string): void {
+            //
+        }
+        detectChanges(): void {
+            //
+        }
```
Comment (InnaAtanasova) [inline libs/platform/src/lib/components/form/combobox/combobox-mobile/combobox/combobox-mobile.component.spec.ts:38]: I'm sorry, but why is this added?

Comment (dimamarksman) [reply libs/platform/src/lib/components/form/combobox/combobox-mobile/combobox/combobox-mobile.component.spec.ts:38]: Particularly here it's just a stub for the specification.
But in general I added this method to the `ComboboxInterface` to be able to run it in another component

**Review by DeepakSap14 — APPROVED** [review]

File: libs/platform/src/lib/components/form/combobox/combobox-mobile/combobox/combobox-mobile.component.spec.ts (hunk 1 above)
Comment (InnaAtanasova) [reply libs/platform/src/lib/components/form/combobox/combobox-mobile/combobox/combobox-mobile.component.spec.ts:38]: Sorry, I meant the empty comments, assuming these are comments `//` :D I see before the blocks were empty.

**Review by InnaAtanasova — APPROVED** [review]
